src/kbmod/search/RawImage.cpp

```diff
@@ -342,36 +342,35 @@ std::vector<float> RawImage::bilinearInterp(float x, float y) const {
     // Top right
     float ax = x + 0.5;
     float ay = y + 0.5;
-    float aPx = floor(ax);
-    float aPy = floor(ay);
-    float aAmount = (ax - aPx) * (ay - aPy);
+    float a_px = floor(ax);
+    float a_py = floor(ay);
+    float a_amt = (ax - a_px) * (ay - a_py);
```

Comment:
potentially the case of local variables not being camel-cased? PEP8 says underscores for both local vars and funcs and camel-case only for Classes. Google C++ style also underscores all local variables, unless they're type names in which case capitalized camel case, and in case they're class members we end them with an extra trailing `_`.